ducktape/tests/runner_client.py

```diff
@@ -157,7 +158,7 @@ def run(self):
     def setup_test(self):
         """start services etc"""
         self.log(logging.INFO, "Setting up...")
-        self.test.setUp()
+        self.test.setup()
```

Comment:
Yeah, I figured that out once I got down to the Test class. See comment re: deprecation given overloads.